### DISCUSSION TITLE
Single click to open file on the explorer pane on the left?

### DISCUSSION THREAD

**Discussion by nsgnkhibdk2cls0f** [stefankueng/BowPad#300] [General]
Currently has to use double click. For other text editors I have ever use, all of them use single click.

**Comment by stefankueng** [comment]
single click would have some issues: right now you can right-click on a file and get it's context menu without the file being opened. With single click opening that wouldn't work.
Also, Windows explorer needs a double click to open a file (if you haven't changed the global setting).

**Reply by nsgnkhibdk2cls0f** [reply]
> single click would have some issues: right now you can right-click on a file and get it's context menu without the file being opened. With single click opening that wouldn't work.

I don't think this is the expected behavior. You can't distinguish between a left click and a right click using code? I tested to set Windows Explorer to use single click using Folder Options. It works as expected: only left single click will open the file, right single click will open the file's context menu but not open the file.
